Simplenote/src/main/java/com/automattic/simplenote/utils/NoteUtils.java

```diff
@@ -75,7 +75,12 @@ public void onClick(DialogInterface dialog, int whichButton) {
 
                             // Show empty placeholder for large devices in landscape.
                             if (activity instanceof NotesActivity) {
-                                ((NotesActivity) activity).showDetailPlaceholder();
+                                NotesActivity notesActivity = (NotesActivity) activity;
+                                if(notesActivity.getNoteListFragment() != null) {
```

Comment:
```suggestion
                                if (notesActivity.getNoteListFragment() != null) {
```

This is missing a space between the `if` and `(`.